.github/workflows/build.yml

```diff
@@ -10,6 +10,7 @@ on:
 
 env:
   config: Release
+  newVersion: 99.99.99-test
```

Comment:
Maybe name this `fakeWebJobsPackageVersion` or something that makes it more clear that this isn't a real version number?

Reply:
done!